utils/src/utils/queries.clj

```diff
@@ -0,0 +1,30 @@
+(ns utils.queries
+  (:require [graphql-builder.parser :refer [parse read-file]]
+            [graphql-builder.core :as core]
+            [jsonista.core :as json]
+            [clojure.string :as str]
+            [clojure.tools.logging :as log]
+            [data.utils :refer [list-files]]))
+
+(defn graphql-file-filter
+  [fname]
+  (re-matches #"[.]graphql" fname))
+
+(def graphql-files
+  (->> (list-files "resources/")
+       (map #(.getName %))
+       #_(filter graphql-file-filter)))
```

Comment:
should this be enabled?

Reply:
Yes it should, but first I will need to fix filter itself